src/github.com/getlantern/flashlight/flashlight.go

```diff
@@ -215,22 +296,19 @@ func runClientProxy(cfg *config.Config) {
 	settings.Configure(cfg, version, buildDate)
 	proxiedsites.Configure(cfg.ProxiedSites)
 
-	hqfd := cfg.Client.HighestQOSFrontedDialer()
-	if hqfd == nil {
-		log.Errorf("No fronted dialer available, not enabling geolocation, stats or analytics")
-	} else {
-		// An *http.Client that uses the highest QOS dialer.
-		hqfdClient := hqfd.NewDirectDomainFronter()
-
-		geolookup.Configure(hqfdClient)
-		statserver.Configure(hqfdClient)
-		analytics.Configure(cfg, false, hqfdClient)
-	}
+	updateClientDirectFronter(cfg, mch)
+	wdc := mch.makeWithClient()
+	geolookup.Configure(wdc)
+	statserver.Configure(wdc)
```

Comment:
statserver does -- it's used for geolookup for *local* stats on peers. I think analytics actually probably breaks with using DDF though.